### PR TITLE
simplify __main__.py

### PR DESCRIPTION
**Pull request by mschrimpf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,152 +1,5 @@
...
-*.code-workspace
-*.nc
-*.zarr
-*.zr
-*.xarray
-*.xr
-cache
-.cache
-.idea/
-wandb/
```
Comment (aalok-sathe) [inline .gitignore:151]: can we keep these things?

File: .gitignore [hunk 2]
```diff
@@ -1,152 +1,5 @@
...
-/site
-
-# mypy
-.mypy_cache/
-.dmypy.json
-dmypy.json
-
-# Pyre type checker
-.pyre/
-*.swp
```
Comment (aalok-sathe) [inline .gitignore:131]: for vim

File: .gitignore [hunk 3]
```diff
@@ -1,152 +1,5 @@
...
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
```
Comment (aalok-sathe) [inline .gitignore:26]: these might all be useful :)

File: README.md [hunk 4]
```diff
@@ -18,6 +18,14 @@ This repository is under active development, and should be considered to be in a
 Code is likely to change substantially, without backwards compatibility.
 Please let us know of any problems by [creating an issue](https://github.com/brain-score/language/issues/new).
 
+## Usage
+
+Score an existing model on an existing benchmark:
+
+```bash
+python brainscore_language score --model_identifier='distilgpt2' --benchmark_identifier='Futrell2018-pearsonr'
```
Comment (aalok-sathe) [inline README.md:26]: 😄

File: brainscore_language/__main__.py [hunk 5]
```diff
@@ -1,350 +1,7 @@
-import argcomplete, argparse
-import brainscore_language as lbs
-import xarray as xr
-import wandb
-import os
+import fire
 
+# noinspection PyUnresolvedReferences
+from . import score
```
Comment (aalok-sathe) [inline brainscore_language/__main__.py:4]: a little uncomfortable here about the relative import :)

Comment (aalok-sathe) [reply brainscore_language/__main__.py:4]: i've just had so many bad experiences not doing a full module-level import
(`from brainscore_language.xyz import score`)

**Review by aalok-sathe — APPROVED** [review]

File: brainscore_language/__main__.py (hunk 5 above)
Comment (mschrimpf) [reply brainscore_language/__main__.py:4]: happy to do absolute here

File: .gitignore (hunk 3 above)
Comment (mschrimpf) [reply .gitignore:26]: might or are? I'd prefer to keep this simple, but if there are specific issues you ran into without these, happy to keep them

File: .gitignore (hunk 2 above)
Comment (mschrimpf) [reply .gitignore:131]: do you want to keep the full list?

**Comment by mschrimpf** [comment]
In the interest of moving this along, I reset the `.gitignore` to how it was. I'll merge the `__main__` changes, and we can revisit the `.gitignore` if need be.

**Pull request merged**